heimdall2/values.yaml

```diff
@@ -166,20 +216,95 @@ heimdall:
     #     secretName: ingress-secret
     #     hosts:
     #       - heimdall.example.com
+  
+  #gateway:
+  #  enabled: false
+  #  apiVersion: 
+  #  kind: VirtualService 
+    
+    #gateways:
+      # List of gateways
+      #hosts:
+        # List of hostsnames that the gateway handles traffic for
+        #- 
+      #http: Route Config
+        #- name: Name for the route group
+          #route: 
+            #- destination:
+                #host: Service name within cluster
+                #port: Port number of service
+                  #number: 3000
+    #annotations:
+      # Optional annotations for the gateway
 
   podAnnotations: {}
+  # Attaches metadata to the Postgre pod
+  # Example:
+    # imageregistry: "https://example.docker.com"
 
   podSecurityContext: {}
+  # Defines privilege and access control settings for the Postgre pod
+  # Example:
+    # runAsUser: 1000
+    # runAsGroup: 3000
+    # fsGroup: 2000
 
   securityContext: {}
+  # Defines privilege and access control settings for the Postgre container
+  # Example:
+    # runAsUser: 1000
+    # runAsGroup: 3000
+    # fsGroup: 2000
 
   resources: {}
+  # Specifies what resources are available to the Postgre container
+  # Example:
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # limits:
+    #   cpu: 200m
+    #   memory: 256Mi
 
   nodeSelector: {}
+  # Selects nodes with specific labels
+  # Example:
+    # kubernetes.io/example: linux
+    # kubernetes.io/example: amd64
 
   tolerations: []
+  # Tolerations allow the scheduler to schedule pods with matching taints
+  # Example:
+    # - key: "node-role.kubernetes.io/example"
+    #   operator: "Exists"
+    #   effect: "NoSchedule"
 
   affinity: {}
-
-# imagePullSecrets:
-#   - name: your-secret-name
+  # Attracts pods to nodes with specific labels
+  # Example:
+    # podAffinity:
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     - weight: 1
+    #         matchExpressions:
+    #           - key: app
+    #             operator: In
+
+#  livenessProbe:
+#    httpGet:
+#      path: /
+#      port: 3000
+#    initialDelaySeconds: 80
+#    periodSeconds: 10
+#    timeoutSeconds: 15
+#    failureThreshold: 3
+
+#  readinessProbe:
+#    httpGet:
+#      path: /
+#      port: 3000
+#    initialDelaySeconds: 80
+#    periodSeconds: 10
+#    timeoutSeconds: 15
+#    failureThreshold: 3
+#    imagePullSecrets:
```

Comment:
this is indented to the wrong level.  it should be set at the very top level, not be underneath readiness probe